server/menus/tests.py

```diff
@@ -11,16 +13,19 @@ def setUp(self):
         self.test_user = User.objects.create(username="Foxy",
                                              first_name="Foxy")
         self.chef = Chef.objects.create(user=self.test_user)
+        self.menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
+        self.item = Item.objects.create(menu=self.menu, title="kibbles", size="the whole bowl",
+                                   price=13.33, description="food for foxy")
 
     def test_menu_creation(self):
         '''
         Makes sure menu is properly created and linked to chef.
         '''
 
         # Tests creation of menu and connection to user
-        self.assertFalse(Menu.objects.filter(chef=self.chef).exists())
-        menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
-        self.assertTrue(Menu.objects.filter(chef=self.chef).exists())
+        self.assertFalse(Menu.objects.filter(title='foxys favorites').exists())
```

Comment:
you don't need to really test this part. I would just start on line 28 with the "assertTrue" that the menu exists. That way you can use the menu from setup (self.menu) (you'll have to change it in all the other places you reference "menu" as well so don't forget that). This test is more about testing the link between menu and item then it is about testing the menu creation. Make sense? (maybe we should rename this test to test_menu_item_link?)